app/controllers/api/v2/base_controller.rb

```diff
@@ -64,6 +66,10 @@ def metadata_total
         @total ||= resource_scope.try(:count).to_i
       end
 
+      def can_create?
+        authorized_for(controller: controller_permission, action: 'create', user_id: User.current, authorizer: authorizer)
```

Comment:
nitpick: shouldnt ```user_id: User.current.id``` ?

Reply:
That's what I though but looking at the code [here](https://github.com/theforeman/foreman/blob/develop/app/helpers/application_helper.rb#L129) it seemed it was expecting `User.current`